app/manifest.json

```
@@ -21,7 +21,8 @@
   },
   "applications": {
     "gecko": {
-      "id": "webextension@metamask.io"
+      "id": "webextension@metamask.io",
+      "strict_min_version": "53"
```

Looks like this might need to be 53.0 (i.e. should match pattern "^[0-9]{1,3}(\.[a-z0-9]+)+$"):

```suggestion
      "strict_min_version": "53.0"
```

---

Looking at the other notices, could we maybe bump this to version 56? It seems like we use a few things from 56.

---

Or if we could set a recommended version?

---

Bumping to 56 makes sense, but I'd rather do that in a subsequent PR. It would involve updating the babel config as well, and it'd be nice to have that concern (Firefox compatibility) in one place. No pressing need anyway, as we've been allowing those versions of Firefox to install our plugin forever.